datastore/longhorn.go

```diff
@@ -3364,6 +3428,24 @@ func (s *DataStore) GetEngineImageCLIAPIVersion(imageName string) (int, error) {
 	return ei.Status.CLIAPIVersion, nil
 }
 
+// GetImageCLIAPIVersion get engine or instance manager image for the given name and returns the CLIAPIVersion
+func (s *DataStore) GetImageCLIAPIVersion(imageName string, backendStoreDriver longhorn.BackendStoreDriverType) (int, error) {
+	if imageName == "" {
+		return -1, fmt.Errorf("cannot check the CLI API Version based on empty image name")
+	}
+
+	if backendStoreDriver == longhorn.BackendStoreDriverTypeV2 {
+		return 0, nil
```

Comment:
We don't need to have version control for v2?

Reply:
v2 volume doesn't need the version for now, so just simply return 0 here.
I think we can add it if needed in the future. WDYT?

Reply:
That's OK. If we can do it backward compatible for new instance manager. That will be perfect and ideal design. 

We should design the communication interface w/o any version control.